scripts/mender.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
   <manifest>
-  <default sync-j="4" revision="rocko"/>
```

Comment:
I guess it is ok to remove this as we typically set the `revision` for each layer. We have done at least so far.

One note about the wording of the commit message, 

      Notably this will not coexist with the Freescale Community BSP

Compulab actually is using the NXP BSP, do not know if the problem would have been the same with the community one.